Mathlib/MeasureTheory/Measure/GiryMonad.lean

```diff
@@ -64,6 +64,24 @@ theorem measurable_measure {μ : α → Measure β} :
     Measurable μ ↔ ∀ (s : Set β), MeasurableSet s → Measurable fun b => μ b s :=
   ⟨fun hμ _s hs => (measurable_coe hs).comp hμ, measurable_of_measurable_coe μ⟩
 
+theorem _root_.Measurable.measure_of_basis {μ : α → Measure β} [∀ a, IsFiniteMeasure (μ a)]
```

Comment:
`basis` is not used anywhere else to refer to that kind of statement if I am not mistaken. How about this:
```suggestion
theorem _root_.Measurable.measure_of_isPiSystem {μ : α → Measure β} [∀ a, IsFiniteMeasure (μ a)]
```
(and same change for the next lemma)